### PR TITLE
feat(LayoutPlugin): allow to set and pass crossOrigin for styles and scripts

### PR DESCRIPTION
**Pull request by artemipanchuk**
Idea is to pass crossOrigin from plugin options to scripts and styles.

### CODE REVIEW

**Review by ValeraS — CHANGES_REQUESTED** [review]

File: src/plugins/layout/index.ts [hunk 1]
```diff
@@ -65,13 +65,14 @@ export function createLayoutPlugin({
                     ...jsAssets.map((js) => ({
                         src: getAbsoluteUrl(publicPath, js, options?.prefix),
                         defer: true,
-                        crossOrigin: 'anonymous' as const,
+                        crossOrigin: options.scriptsCrossOrigin || 'anonymous',
```
[the remark this answers is not in the document]
Comment (artemipanchuk) [reply src/plugins/layout/index.ts:68]: Yes. I left "anonymous" as default. But for consistency added scriptsCrossOrigin as possible option.

[the remark this answers is not in the document]
Comment (artemipanchuk) [reply src/plugins/layout/index.ts:68]: I don't know where else crossOrigin may be needed. We encountered situation, that we need it only for stylesheets.

[the remark this answers is not in the document]
Comment (ValeraS) [reply src/plugins/layout/index.ts:68]: I mean here https://github.com/gravity-ui/app-layout/blob/main/src/plugins/layout/index.ts#L82-L95

[the remark this answers is not in the document]
Comment (artemipanchuk) [reply src/plugins/layout/index.ts:68]: Ok, fair enough, i've added the same settings for this case

**Pull request merged**